lib/challenge.js

```diff
@@ -30,10 +40,14 @@
   // - a name
   // - an address
 
-var pet_owner;
 
-// Answer Starts Here
 
+// Answer Starts Here
+var pet_owner {
+  clinic:
+  owner: [name, address]
```

Comment:
For an object representing a pet owner, who is, presumably, a human being, to have an attribute called `owner` suggests that our pet-owning human being has an owner, doesn't it?

A suggestion on lists (arrays): JS will not constrain you as to what types you place in lists, but practicality will. If you're placing values in a list, you usually want to be able to iterate over it, running the same code against every item in the list. For example, if you had a list of people to send a greeting card to, you could go through the list on by one and purchase a greeting card for each of them. Now, let's say you added an odd one out: "bake bread". You'd have to know that "bake bread" is not a name, and that you can't apply the same procedure for that value.

When building a list, build it with this in mind. You can iterate over a list of names, or a list of addresses, but a list of names and addresses would mean your loop logic would need to become more complex. In software development, you want to avoid complexity wherever possible.